scripts/update-e2e-cluster.sh

```diff
@@ -1,131 +0,0 @@
-#!/bin/bash
```

Comment:
This file is not used anymore? Do you want to update README then?

Reply:
Yes, probably could clean up more as well. I just updated the readme